packages/atom/src/types.ts

```diff
@@ -0,0 +1,112 @@
+import { Stream } from '@effection/subscription';
```

Comment:
Even though this is a core package, should we import from `effection`? Feels like when we deprecate subscription, it'll be one less point of coupling. However, I can also see that in order to avoid circular dependencies we could have a policy that core package depend on core packages explicitly.

Or maybe the rule is "blessed packages should depend directly on other blessed packages", which in this case, means that atom could pull from the omnibus `effection` package since it is not a blessed package (and is unlikely to be one)

Reply:
Actually in this PR I did add `atom` as a blessed package, but maybe it shouldn't be?

IMO all packages in the monorepo should depend on the individual subpackages, regardless of whether they are blessed or not.

Reply:
I'd say that this is not a blessed package as you will only need it a certain amount of time that is likely to be < 50%

👍🏻 on the policy of importing from individual packages within the repo.